internal/backends/testing/lib.rs

```diff
@@ -134,8 +134,8 @@ impl PlatformWindow for TestingWindow {
         unimplemented!()
     }
 
-    fn set_clipboard_text(&self, text: String) {
-        *self.backend.clipboard.lock().unwrap() = Some(text);
+    fn set_clipboard_text(&self, text: &str) {
+        *self.backend.clipboard.lock().unwrap() = Some(text.into());
     }
 
     fn clipboard_text(&self) -> Option<String> {
```

Comment:
And what about this one, should it return an `Option<SharedString>`?

Reply:
I'd stay with a `String` as chances are I think highest the platform specific code gets a `String` from somewhere. On the receiving end we're going to throw it away, so if it were `SharedString` there'd be one unnecessary allocation. But no strong feeling about it - if you prefer `SharedString` then I can change it.